automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookGCEDataSyncingSpec.scala

```diff
@@ -99,7 +99,7 @@ class NotebookGCEDataSyncingSpec extends RuntimeFixtureSpec with NotebookTestUti
               logger.info(s"[playground mode] original local content is ${originalLocalContent}")
               val originalLocalContentSize: Int = originalLocalContent.size
 
-              originalRemoteContentSize shouldBe originalLocalContentSize
+              abs(originalRemoteContentSize - originalLocalContentSize)<2 shouldBe true
```

Comment:
I think in ScalaTest you can do:
```
originalRemoteContentSize shouldBe originalLocalContentSize +- 1
```

https://www.scalatest.org/user_guide/using_matchers